components/NewsItem.vue

```diff
@@ -1,18 +1,19 @@
 <template>
 	<!-- This compenent is used by data from pages api call as well as post api call -->
 	<article>
-		<figure class="preview-media">
+		<figure v-if="showPreviewImage" class="preview-media">
 			<img v-if="picture" :src="picture" :alt="altTag" class="news-item-img">
 			<WeCountLogo v-else class="news-item-img" />
 		</figure>
 		<h2 class="h3">
-			<nuxt-link :to="'/news-and-views/' + slug">
-				{{ title }}
-			</nuxt-link>
+			<a v-if="isExternalHref" :href="href" v-html="title" target="_blank" />
```

Comment:
I don't think we should use `target="_blank"`. See:

- https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/links/links-and-new-windows/
- https://css-tricks.com/use-target_blank/